lib/colorls/core.rb

```diff
@@ -257,7 +257,7 @@ def git_dir_info(path)
     def long_info(content)
       return '' unless @long
 
-      [mode_info(content.stats), content.nlink, user_info(content), group_info(content.group),
+      [mode_info(content.stats), content.nlink.to_s.rjust(4, ' '), user_info(content), group_info(content.group),
```

Comment:
It seems wasteful to always use 4 characters for this column. It probably looks weird when all files the listing have only a one-digit number of links.

It would be better to determine the maximum of the list and use that as the columns size, this would also work when `nlink > 9999`.